README.md

```diff
@@ -55,12 +55,11 @@ The Tetrapoda.owl example (same procedure: load into Protege, run reasoner, swit
   TU and has_Ancestor some (has_Progenitor some ('has part' some 'manual digit'))
   ```
 
-IRIs, Deployment, and versioning
---------------------------------
+IRIs, deployment and versioning
+-------------------------------
 
-The Phyloreferencing Ontology ([`phyloref.owl`](./phyloref.owl)) uses as the base IRI `http://ontology.phyloref.org/phyloref.owl`. The demonstration A-box ontologies use `http://ontology.phyloref.org/demo/` as prefix.
+The Phyloreferencing Ontology uses as its base IRI `http://ontology.phyloref.org/phyloref.owl`, which is served by GitHub Pages from [`docs/phyloref.owl`](./docs/phyloref.owl). This file is only updated upon a new release of this ontology. Thus, it may not be identical to [`./phyloref.owl`](./phyloref.owl), whose latest development version can be found in the `master` branch. We recommend using the canonical IRI.
 
-These IRIs currently resolve to content in this repository, served via GitHub Pages from the `docs/` directory. Content in that directory is currently not updated automatically, and only upon releases.
-Thus the version of the ontology available via the canocnical IRIs may not be the same as the latest version in the `master` branch of this repository.
+The demonstration A-box ontologies use `http://ontology.phyloref.org/demo/` as their prefix, but can be imported from `http://ontology.phyloref.org/[Demo Name].owl`. A full list of demonstration ontologies can be found in the [`docs/`](./docs) directory.
```

Comment:
I'm wondering whether it wouldn't be better to change the demonstration files to add `.owl` to the corresponding URIs/IRIs. I did mean those to be resolvable, but overlooked that the IRIs left out the .owl extension. Is there a good reason not to simply fix that?

Reply:
To make them resolvable, we'd need to add `.owl` to the demonstration file IRIs (744fcff, 7e57d8f) and to move them into the `docs/demo/` directory (4410c82) so that they resolve at e.g. `http://ontology.phyloref.org/demo/Campanulaceae.owl`. I've just done that, so they should be resolvable now!

Reply:
Thanks for doing that 😃 I actually meant to put them into the `docs/demo/` folder, another oversight.

BTW I wasn't expecting you to fix up my mistakes, it would have been fine to just point what was missing. Nonetheless, appreciate the help 😄

Reply:
So now we need to change the README back, right?

Reply:
No worries -- I was going to make a list of necessary changes, but then I figured it'd be easier to just write up some commits!